CHANGELOG.unreleased.md

```diff
@@ -13,7 +13,7 @@
 
 ### Bug fixes
 
-> Users must be able to say: “I had this issue, happy to know it's fixed”
+- [Resource Set] Fix issue that caused the `resourceSet.removeSubject` rpc call to fail
```

Comment:
`XO` users use `xo-cli` or UI instead of `RPC calls` to communicate with `xo-server`. So, i suggest:

```suggestion
> Users must be able to say: “I had this issue, happy to know it's fixed”

- [Resource Set] Fix `this.removeSubjectFromResourceSet is not a function` error on calling `resourceSet.removeSubject` via `xo-cli` [#5265](https://github.com/vatesfr/xen-orchestra/issues/5265) (PR [#5266](https://github.com/vatesfr/xen-orchestra/pull/5266))
```